packages/frontend/core/src/components/pure/cmdk/data.tsx

```diff
@@ -301,6 +314,18 @@ export const usePageCommands = () => {
 
       // check if the pages have exact match. if not, we should show the "create page" command
       if (results.every(command => command.originalValue !== query)) {
+        results.push({
+          id: 'affine:pages:append-to-journal',
+          label: t['com.affine.journal.cmdk.append-to-today'](),
+          value: 'affine::append-journal' + query, // hack to make the page always showing in the search result
+          category: 'affine:creation',
+          run: async () => {
+            await journalHelper.appendContentToToday(query);
```

Comment:
Focus the new block after being appended

Reply:
🤔 The route may change here,  is it possible to focus after the route jump?

Reply:
I think you can get both the new the block id plus the page id. So instead of using `openToday`, you can use jump to page instead

Reply:
Modified!